app/views/posts/show.html.erb

```diff
@@ -1 +1,18 @@
-<h1>Here is the list of posts</h1>
\ No newline at end of file
+<div class="wrapper">
+  <div class="border post">
+    <div class="post-header">
+      <h3>Post#<%=@post.id%> by <%= @user.name%></h3>
+      <ul class="comments-likes">
+        <li>Comments: <%=@post.comments_counter%>,</li>
+        <li>Likes: <%=@post.likes_counter%>,</li>
+      </ul>
+    </div>
+     <p><%= @post.text %></p>
+  </div>
+    <ul class="list-type-none border comments">
+      <% @comments.each do |comment| %>
+        <li> <%= comment.user.name %> : <%= comment.text %></li>
+      <% end %>
+    </ul>
+  </div>
+</div>
```

Comment:
- [ ] As required for the project, Kindly ensure that the Post show page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png). For that, I also suggest when a user clicks on a `Post` it redirects to its `show` page. 
Right now, if the user opens the `Post show page`, your app displays the below error. **Make sure you check how you implemented the `Show` action from your `PostsController`**, change in your views where you use `user` keyword to `author` as `comments` and `posts` belong to an `author` and then fix this error :point_down: :point_down: 
![Screenshot from 2023-03-17 19-27-09](https://user-images.githubusercontent.com/83042742/225976079-7c05b3a0-7794-4927-9e5e-d6a7b1e880a8.png)